roboflow/util/prediction.py

```diff
@@ -367,6 +389,17 @@ def save(self, output_path="predictions.jpg", stroke=2):
                     (0, 0, 0),
                     1,
                 )
+            elif self.base_prediction_type == INSTANCE_SEGMENTATION_MODEL:
+                points = [[int(p["x"]), int(p["y"])] for p in prediction["points"]]
+                np_points = np.array(points, dtype=np.int32)
+                cv2.polylines(
+                    image,
+                    [np_points],
+                    isClosed=True,
+                    color=stroke_color,
+                    thickness=stroke,
+                )
```

Comment:
There's a lot of duplicated image logic between `Prediction` and `PredictionGroup`. We should refactor this to reuse the same logic between the two classes.

Reply:
Definitely